awx/settings/defaults.py

```diff
@@ -477,10 +477,6 @@ def IS_TESTING(argv=None):
         'schedule': timedelta(seconds=60),
         'options': {'expires': 50,}
     },
-    'purge_stdout_files': {
```

Comment:
We do still need this.

Reply:
@rooftopcellist also is looking to put in a schedule for cleaning up expired tokens.